_includes/footer.html

```diff
@@ -12,10 +12,7 @@ <h2 class="footer-heading">{{ site.title | escape }}</h2>
       <div class="footer-col footer-col-2">
         <ul class="social-media-list" style="line-height: 1.8rem">
           <li>
-            <a href="https://bitbucket.org/vitalab/vitalabai_public">Bitbucket</a>
-          </li>
-          <li>
-            <a href="http://vital.dinf.usherbrooke.ca/">Website</a>
+            <a href="https://github.com/vitalab/vitalab.github.io">gitHub</a>
```

Comment:
Wrong capitalization:
```suggestion
            <a href="https://github.com/vitalab/vitalab.github.io">GitHub</a>
```